src/TwilioSmsMessage.php

```diff
@@ -158,4 +167,30 @@ public function validityPeriod(int $validityPeriodSeconds): self
 
         return $this;
     }
+
+    /**
+     * Set the attempt option.
```

Comment:
Can you add comments here and to the other one explaining what this does?

Reply:
Sure! I hope this clarifies it